src/panels/lovelace/editor/config-elements/hui-tile-card-editor.ts

```diff
@@ -27,6 +41,7 @@ const cardConfigStruct = assign(
     show_entity_picture: optional(boolean()),
     tap_action: optional(actionConfigStruct),
     icon_tap_action: optional(actionConfigStruct),
+    extras: optional(array(any())),
```

Comment:
let's add structs (can be in a new PR)

Reply:
There can be custom extra so we can not check this here. May be in the extra editor?